.dotnet/Azure.AI.OpenAI.Staging/Custom/Audio/AzureAudioClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenAI.Audio;
+using System.ClientModel.Primitives;
+
+namespace Azure.AI.OpenAI.Staging.Audio;
+
+/// <summary>
+/// The scenario client used for audio operations with the Azure OpenAI service.
+/// </summary>
+/// <remarks>
+/// To retrieve an instance of this type, use the matching method on <see cref="AzureOpenAIClient"/>.
+/// </remarks>
+public partial class AzureAudioClient : AudioClient
```

Comment:
Curious: what motivates making these public?

Reply:
Egads, I missed some! They're not intended to be anymore.

It might be nice to not have to create the top-level client, but we agreed we'd consider that later since it's a non-disruptive, additive change.